storage/innobase/xtrabackup/src/backup_mysql.cc

```diff
@@ -171,6 +171,17 @@ MYSQL *xb_mysql_connect() {
 
   xb_mysql_query(connection, "SET SESSION wait_timeout=2147483", false, true);
 
+  if (xb_mysql_numrows(connection, "SELECT @@wsrep_sync_wait", false) > 0) {
+    xb_mysql_query(connection, "SET SESSION wsrep_sync_wait=0", false, true);
+  }
+
+  if (xb_mysql_numrows(connection, "SELECT @@group-replication-consistency",
```

Comment:
Same here, have a var for GR enabled that we check when we do the initial SHOW GLOBAL VARIABLES and use this new var here.

Reply:
I could not find var for GR. Also same reason previous commit.